apps/Notes Up.md

```diff
@@ -32,8 +32,8 @@ screenshots:
 installation:
   - system: elementary
     info:
-    - version: '0.3'
-      repository: 'ppa:philip.scott/projects'
+    - version: '1.3'
```

Comment:
Version refers to the OS version the package / PPA applies to, so it should still be 0.3 for Freya and (an extra entry) / or 0.4 for Loki.